perfmetrics/scripts/testing_on_gke/examples/dlio/parse_logs.py

```diff
@@ -26,7 +26,7 @@
 # local library imports
 sys.path.append("../")
 import dlio_workload
-from utils.utils import get_memory, get_cpu, standard_timestamp, is_mash_installed
+from utils.utils import get_memory, get_cpu, unix_to_timestamp, standard_timestamp, is_mash_installed, get_memory_from_monitoring_api, get_cpu_from_monitoring_api, timestamp_to_epoch
 from utils.parse_logs_common import ensureDir, download_gcs_objects, parseLogParserArguments, SUPPORTED_SCENARIOS
```

Comment:
Rename ensureDir, parseLogParserArguments to use `_` in place of camel-case for consistency.

Reply:
will do in #2496